src/main/java/com/thebois/views/StructureView.java

```diff
@@ -0,0 +1,56 @@
+package com.thebois.views;
+
+import java.util.ArrayList;
+
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.glutils.ShapeRenderer;
+
+import com.thebois.models.world.structures.IStructure;
+
+/**
+ * Responsible for displaying structures.
+ */
+public final class StructureView implements IView {
+
+    private final float tileSize;
+    private ArrayList<IStructure> structures = new ArrayList<>();
+    private final Color brown = new Color(0.4f, 0.2f, 0f, 1);
+
+    /**
+     * Creates an instance of a Structure view.
+     *
+     * @param tileSize The tile size of the world in order to render structures in the right size.
```

Comment:
Not sure if the reason why it is needed should be here. I think "The size of a single world tile in screen space" would fit better.

Reply:
(The reason I'm being so picky about wording is to reduce confusion in the future when we need to look back at and understand what we meant today.)

Reply:
It's good that you are picky, hopefully, I learn and do better next time.